tuxemon/core/components/event/actions/player.py

```diff
@@ -32,6 +32,7 @@
 from core import tools
 from core.components import item
 from core.components import monster
+import Common
```

Comment:
This should be changed to:
```python
from core.components.event.actions.common import Common
```

Otherwise the game crashes on launch:
```python
Traceback (most recent call last):
  File "./tuxemon.py", line 60, in <module>
    main()
  File "/tmp/xirsoi/Tuxemon/tuxemon/core/main.py", line 57, in main
    control = PygameControl(prepare.ORIGINAL_CAPTION)
  File "/tmp/xirsoi/Tuxemon/tuxemon/core/control.py", line 75, in __init__
    self.event_engine = event.EventEngine()
  File "/tmp/xirsoi/Tuxemon/tuxemon/core/components/event/__init__.py", line 73, in __init__
    self.actions = plugin.get_available_methods(action_plugins)
  File "/tmp/xirsoi/Tuxemon/tuxemon/core/components/plugin.py", line 148, in get_available_methods
    for plugin in plugin_manager.getAllPlugins():
  File "/tmp/xirsoi/Tuxemon/tuxemon/core/components/plugin.py", line 93, in getAllPlugins
    m = importlib.import_module(module)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/tmp/xirsoi/Tuxemon/tuxemon/core/components/event/actions/player.py", line 35, in <module>
    import Common
ImportError: No module named Common
```